source/gui/guiHelper.py

```diff
@@ -138,9 +139,21 @@ def addButton(self, *args, **kwargs):
 )
 
 
+@overload
+def associateElements(firstElement: wx.StaticText, secondElement: _HorizontalCtrlT) -> wx.BoxSizer: ...
+@overload
+def associateElements(firstElement: wx.StaticText, secondElement: wx.CheckBox) -> wx.BoxSizer: ...
+@overload
+def associateElements(firstElement: wx.StaticText, secondElement: _VerticalCtrlT) -> wx.BoxSizer: ...
+@overload
+def associateElements(firstElement: wx.Button, secondElement: wx.CheckBox) -> wx.BoxSizer: ...
+@overload
+def associateElements(firstElement: wx.TextCtrl, secondElement: wx.Button) -> wx.BoxSizer: ...
+
+
 def associateElements(
-	firstElement: wx.StaticText | wx.StaticText | wx.StaticText | wx.Button | wx.TextCtrl,
-	secondElement: _HorizontalCtrlT | wx.CheckBox | _VerticalCtrlT | wx.CheckBox | wx.Button,
+	firstElement: wx.StaticText | wx.Button | wx.TextCtrl,
+	secondElement: _HorizontalCtrlT | wx.CheckBox | _VerticalCtrlT | wx.Button,
 ) -> wx.BoxSizer:
```

Comment:
The [docs](https://docs.python.org/3/library/typing.html#overload) and [PEP](https://peps.python.org/pep-0484/#function-method-overloading) both suggest the actual implementation shouldn't have type annotations, since they're defined above.

Reply:
Good catch - copilot misled me